eng/common/pipelines/templates/steps/emit-rate-limit-metrics.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  - name: GitHubUser
+    type: string
+  - name: GitHubToken
+    type: string
```

Comment:
Should we also add some context of where we are logging from? So we can start adding this to various places to see what the limits look like in different contexts?

Reply:
That may already be included in the metric metadata as we process the build info at the same time

Reply:
We'd be applying false granularity to the metric.   It'd be context around the job that's checking the metric while the metric being checked is more broad than that context.